home/templates/home/team_detail.html

```diff
@@ -0,0 +1,132 @@
+{% extends "base.html" %}
+{% load i18n static %}
+
+{% block title %}{{ team.name }} - {{ team.session.title }} | Djangonaut Space{% endblock %}
+{% block meta_title %}{{ team.name }} - {{ team.session.title }} | Djangonaut Space{% endblock %}
+
+{% block content %}
+<main class="section my-3 mx-5">
+  <div class="section-container max-w-6xl mx-auto">
+    <nav class="mb-4 text-sm" aria-label="breadcrumb">
+      <ol class="flex items-center gap-2">
+        <li><a href="{% url 'user_sessions' %}" class="text-blue-600 hover:text-blue-800 hover:underline">{% trans "My Sessions" %}</a></li>
+        <li class="text-gray-400">/</li>
+        <li class="text-gray-600">{{ team.session.title }} - {{ team.name }}</li>
+      </ol>
+    </nav>
+
+    <h1 class="text-5xl mb-4">{{ team.name }}</h1>
+
+    <div class="mb-6">
+      {% if team.session.status == 'current' %}
+        <span class="inline-block bg-purple-100 text-purple-800 px-3 py-1 rounded-full text-sm font-bold ml-2">
+          Active
+          {% if team.session.current_week %}
+            - Week {{ team.session.current_week }}
+          {% endif %}
```

Comment:
```suggestion
          Active - Week {{ team.session.current_week }}
```
Revise current_week to show negative values.